src/config/settings.py

```
@@ -0,0 +1,72 @@
+    PATH_TO_MOVIES_CSV: str = str(BASE_DIR / "database" / "seed_data" / "imdb_movies.csv")
+
+    PATH_TO_EMAIL_TEMPLATES_DIR: str = str(BASE_DIR / "notifications" / "templates")
+    ACTIVATION_EMAIL_TEMPLATE_NAME: str = "activation_request.html"
+    ACTIVATION_COMPLETE_EMAIL_TEMPLATE_NAME: str = "activation_complete.html"
+    PASSWORD_RESET_TEMPLATE_NAME: str = "password_reset_request.html"
+    PASSWORD_RESET_COMPLETE_TEMPLATE_NAME: str = "password_reset_complete.html"
+
+    LOGIN_TIME_DAYS: int = 7
+
+    EMAIL_HOST: str = os.getenv("EMAIL_HOST", "host")
+    EMAIL_PORT: int = int(os.getenv("EMAIL_PORT", 25))
```

Pylint (3.3.7)

[warning] os.getenv default type is builtins.int. Expected str or None.

(W1508)